wordle-web/src/components/AppBar.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <v-card>
+    <nav>
+      <v-layout>
+        <v-toolbar prominent>
+          <RouterLink to="/"><icon /></RouterLink>
```

Comment:
You should wrap both the icon and the title in the same router link

Reply:
I'm working on that, the issue is it keeps messing with my formatting of both the link and the rest of the bar.  Once I get it figured out, I will make sure they are the same link.